babel.config.json

```diff
@@ -4,5 +4,14 @@
   ],
   "plugins": [
     ["angularjs-annotate", { "es6": true, "dynamicImport": true, "explicitOnly": false }]
+  ],
+  "ignore": [
+    "node_modules/esn-frontend-common-libs/src/frontend/components/ical.js/build/*.js",
+    "node_modules/esn-frontend-common-libs/node_modules/restangular/**/*.js",
+    "node_modules/restangular/**/*.js",
+    "node_modules/@linagora/ical.js/**/*.js",
+    "node_modules/angular/angular.js",
+    "node_modules/fullcalendar/dist/fullcalendar.js",
+    "node_modules/angular-material/angular-material.js"
```

Comment:
Do we need this now that we don't use Babel for tests?

Reply:
No we don't, just forgot to remove this. Thanks!